test/climber/README.md

```diff
@@ -0,0 +1,1034 @@
+...
+if (getOperationState(id) != OperationState.ReadyForExecution) {
+    revert NotReadyForExecution(id);
+}
+```
+Even if we find a way to somehow register the `operation` (reminding: `attacker` is neither an `admin` nor a `proposer`) - we still need to fulfill two conditions to properly prepare the test for symbolic execution:
+1. Bypass symbolic offset error in `execute()`. The "antidote" to it is to use the `CreateCalldata()` cheatcode to replace `dataElements[]`. 
+2. `dataElements[]` bytes in the `schedule()` should be replaced not via `CreateCalldata()`, but via `CreateBytes()` cheatcode. The thing is, this seems more correct: we can really pass literally any bytes here. This is especially true in setups with upgradable proxy (nothing prevents us from creating an `operation` for proxy, which is not supported by the current **implementation** yet, but maybe someday it will be). And using `CreateCalldata()` imposes restrictions on what these bytes can be. Although I admit that this take can be debatable.
+
+Such a mismatch in different cheatcodes in `schedule()` and `execute()` can badly affect the logic of finding the `id`. The fact is that the bytes generated by `CreateCalldata()` can be of completely different lengths, while `CreateBytes()` requires us to clearly specify the number of generated symbolic bytes ("extra" bytes will be replaced with `0s` as padding).
+
+Now let's remember that we need to preserve the identity of the parameters of both functions to correctly schedule and execute the `operation`. Thus, we will not be able to simply prove that the `operation` we passed to `execute()` was scheduled. We are faced with an unusual problem that needs to be solved.
+
+Here are some ideas on how we can solve this:
+1. Somehow modify symbolic bytes from `CreateCalldata()` so that they are also of static size and output `0s` at the end as padding.
+2. Handle separately situations with potential proxy **implementation** change.
+3. Refactor the `operations` functionality itself to make it more "friendly" for symbolic analysis (Say "hello" to symbolic mapping keys from [backdoor](https://github.com/igorganich/damn-vulnerable-defi-halmos/tree/master/test/backdoor#ownerisnotabeneficiary-issue)).
+
+All of them have the right to exist and can potentially be used to solve this problem. However, they require a lot of work. It is easier to slightly change the principle by which `id` is calculated: instead of concatenating the entire array of bytes, we can try to concatenate only the function selector from each byte array. This selector is always of static size, `4` bytes, which will allow us to check at `execute()` whether such a set of addresses and corresponding functions has been registered (without specific parameters for these functions). Of course, we take into account that any simplification of verification may lead to false counterexamples. But, in my opinion, this should be a profitable trade-off.
```

Comment:
> Of course, we take into account that any simplification of verification may lead to false counterexamples

I was going to say, isn't this going to lead to counterexamples? I guess I'll read on to see how you handle this haha